src/library/basetracktablemodel.cpp

```diff
@@ -639,7 +642,10 @@ QVariant BaseTrackTableModel::roleValue(
                 return QVariant();
             }
             DEBUG_ASSERT(lastPlayedAt.timeSpec() == Qt::UTC);
-            return mixxx::localDateTimeFromUtc(lastPlayedAt);
+            if (role == Qt::ToolTipRole || role == kDataExportRole) {
+                return mixxx::localDateTimeFromUtc(lastPlayedAt);
```

Comment:
Same here, even if it is only a single function call. Especially conversions could get easily out of sync, don't duplicate such code.